karabo_data/geometry2.py

```diff
@@ -145,6 +146,8 @@ class DetectorGeometryBase:
     pixel_size = 0.0
     frag_ss_pixels = 0
     frag_fs_pixels = 0
+    n_modules = 0
+    n_asics_per_module = 0
```

Comment:
I'm generally moving towards using the XFEL terminology ('tiles' instead of 'asics') in this code. Some of the CrystFEL geometry code will continue to match the CrystFEL terms, but for this new attribute, I think it's best to use 'tiles'.

('asics' may also be somewhat inaccurate; I think I saw that the tiles we position for DSSC actually contain multiple ASICs)

Reply:
so n_modules and n_tiles_per_module?

Reply:
Yup :-)

Reply:
Done!